### PR TITLE
docs: Added documentation that the processor name can be set in the worker's Cli option.

### PR DESCRIPTION
**Pull request by kancers**
Added documentation that the processor name can be set in the worker's Cli option. https://github.com/cakephp/queue/pull/51

### CODE REVIEW

**Comment by kancers** [comment]
@markstory @ADmad 
Updated the documentation. Thank you as always

File: docs/en/index.rst [hunk 1]
```diff
@@ -237,6 +237,7 @@ This shell can take a few different options:
 
 - ``--config`` (default: default): Name of a queue config to use
 - ``--queue`` (default: default): Name of queue to bind to
+- ``--processor`` (default: ``null``): Name of processor to bind to
```
Comment (othercorey) [inline docs/en/index.rst:240]: It looks like we use `null` or other defaults, but not sure it makes sense. What is `null` max runtime, for example.

Comment (markstory) [reply docs/en/index.rst:240]: I suspect it means use the default defined in enqueue.

Comment (kancers) [reply docs/en/index.rst:240]: Thank you, both of you.

**Pull request merged**